asana-auto-expand-subtasks.js

```diff
@@ -0,0 +1,32 @@
+// ==UserScript==
+// @name         Asana: Load More Subtasks
+// @namespace    http://tampermonkey.net/
+// @version      0.0.1
+// @description  Automatically expands the "Load more subtasks" section.
+// @author       tbarri
+// @match        *://app.asana.com/*
+// @icon         https://www.google.com/s2/favicons?sz=64&domain=asana.com
+// @grant        none
+// @downloadURL  https://raw.githubusercontent.com/jpbochi/user-scripts/master/asana-auto-expand-subtasks.js
+// @updateURL    https://raw.githubusercontent.com/jpbochi/user-scripts/master/asana-auto-expand-subtasks.js
+// ==/UserScript==
+
+(function() {
+    'use strict';
+
+    function install() {
+        const observer = new MutationObserver((_mutations) => {
+            const buttons = document.getElementsByClassName('HighlightSol HighlightSol--core SubtaskGrid-loadMore PrimaryLinkButton');
+
+            if (buttons.length == 1 && buttons[0].textContent.match(/^load more subtasks$/i)) {
+                console.log("Auto expanding subtasks!")
+                buttons[0].click()
```

Comment:
I'd do an `observer.disconnect();` after this so that we don't keep looking for the button on every change to the body.

Reply:
right! I solved this issue on https://github.com/jpbochi/user-scripts/blob/91afc2afc3d8f9dbc73f6a51f9f7a958ba939d1a/asana-helpers-markdown.js#L107-L113

You can listen to `window.navigation.addEventListener('navigate'…`, and use that `waitForAddedNode` function. This is battle tested by me. I know it works on Vivaldi, at least.

Reply:
it's likely fine. It's better to have something working anyways. Let's merge this.

I may try `window.addEventListener("popstate"` from https://developer.mozilla.org/en-US/docs/Web/API/History_API later.